src/routes/projects/[project]/+page.server.ts

```diff
@@ -0,0 +1,34 @@
+import { regionUrl } from '$lib/url';
```

Comment:
moved from region page as we can add region in the project level. in our case strategise + specific region pages